linux_os/guide/system/permissions/files/rootfiles/rootfiles_configured/oval/shared.xml

```diff
@@ -0,0 +1,30 @@
+{{%- set files = ["bash_logout", "bash_profile", "bashrc", "cshrc", "tcshrc", ] %}}
+
+<def-group>
+    <definition class="compliance" id="{{{ rule_id }}}" version="1">
+        {{{ oval_metadata("Ensure that tmpfiles for rootfiles is configured correctly.") }}}]
+        <criteria operator="AND" comment="Ensure that">
+            {{%- for file in files %}}
+            <criterion comment="Check that /root/.{{{ file }}} is configured correctly"
+                       test_ref="test_{{{ rule_id }}}_{{{ file }}}"/>
+            {{%- endfor %}}
+        </criteria>
+    </definition>
+    {{%- for file in files %}}
+        <ind:textfilecontent54_test check="all"
+                                    comment="Tests that .{{{ file }}} is configured correctly."
+                                    id="test_{{{ rule_id }}}_{{{ file }}}" version="1">
+
+            <ind:object object_ref="obj_{{{ rule_id }}}_{{{ file }}}"/>
+        </ind:textfilecontent54_test>
+        <ind:textfilecontent54_object id="obj_{{{ rule_id }}}_{{{ file }}}" version="1">
+            <ind:path>/etc/tmpfiles.d/</ind:path>
+            <ind:filename operation="pattern match">^.*\.conf$</ind:filename>
+            <ind:pattern operation="pattern match">^C[[:blank:]]+\/root\/.{{{ file
+                }}}[[:blank:]]+600[[:blank:]]+root[[:blank:]]+root[[:blank:]]+-[[:blank:]]+\/usr\/share\/rootfiles/.{{{ file
+                }}}$
+            </ind:pattern>
```

Comment:
All automatus tests fail. It's caused by the newline and spaces between the `$` and `</ind:pattern>`.